Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    print('The time delay distance is {}.'.format(ddt))
</pre><p>do you want to add the units Mpc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Done, thanks.</p>